data/campaigns/Dead_Water/scenarios/05_Tirigaz.cfg

```diff
@@ -696,38 +696,51 @@
         [filter]
             id=Marg-Tonz
         [/filter]
-
-        # The chest looks better when it's offset 10px to the north
-        [item]
-            x,y=$x1,$y1
-            image=misc/blank-hex.png~BLIT(items/chest-open.png~CROP(0,10,72,62),0,0)
-        [/item]
-
         [if]
             [have_unit]
                 x,y=$x2,$y2
-                race=bats
+                side=1
             [/have_unit]
-
+            
             [then]
-                [message]
-                    speaker=second_unit
-                    message= _ "Neep, neep!"
-                [/message]
+                # The chest looks better when it's offset 10px to the north
+                [item]
+                    x,y=$x1,$y1
+                    image=misc/blank-hex.png~BLIT(items/chest-open.png~CROP(0,10,72,62),0,0)
+                [/item]
+        
+                [if]
+                    [have_unit]
+                        x,y=$x2,$y2
+                        race=bats
+                    [/have_unit]
 
-                [message]
-                    speaker=Kai Krellis
-                    message= _ "It seems that orc was rich! He has a chest here with over 100 pieces of gold!"
-                [/message]
+                    [then]
+                        [message]
+                            speaker=second_unit
+                            message= _ "Neep, neep!"
+                        [/message]
+
+                        [message]
+                            speaker=Kai Krellis
+                            message= _ "It seems that orc was rich! He has a chest here with over 100 pieces of gold!"
+                        [/message]
+                    [/then]
+                    [else]
+                        [message]
+                            speaker=second_unit
+                            message= _ "It seems that orc was rich! He has a chest here with over 100 pieces of gold!"
+                        [/message]
+                    [/else]
+                [/if]
             [/then]
             [else]
                 [message]
-                    speaker=second_unit
-                    message= _ "It seems that orc was rich! He has a chest here with over 100 pieces of gold!"
+                    speaker=Kai Krellis
+                    message= _ "Now we must defeat the undead."
```

Comment:
This is a string change - meaning this can not go into 1.14.2 and must wait for 1.14.3. However, you could exclude this message for now, and add it in a new PR when 1.14.2 is out.
To update this PR, you can use
```
git commit --amend after your changes
git push -f
```
That said, I do not know when 1.14.2 will be tagged, and if there is enough time until then.